### PR TITLE
Document the confirm option on the generated invoke APIs

### PR DESCRIPTION
**Pull request by peterbroadhurst**
- The description was missing on the generated `/invoke` APIs
- The description incorrectly referred to a "message" on the base `/contracts/invoke` API

### CODE REVIEW

File: doc-site/docs/swagger/swagger.yaml [hunk 1]
```diff
@@ -5069,7 +5069,8 @@ paths:
       description: Invokes a method on a smart contract. Performs a blockchain transaction.
       operationId: postContractInvoke
       parameters:
-      - description: When true the HTTP request blocks until the message is confirmed
+      - description: When true the HTTP request blocks until the blockchain transaction
+          is confirmed
```
Comment (EnriqueL8) [inline doc-site/docs/swagger/swagger.yaml:5073]: Not sure why there is an end line here?

**Review by EnriqueL8 — APPROVED** [review]
LGTM - one question on the swagger that might be fine

File: doc-site/docs/swagger/swagger.yaml (hunk 1 above)
Comment (peterbroadhurst) [reply doc-site/docs/swagger/swagger.yaml:5073]: Sorry @EnriqueL8 - I'm being slow. What's the problem with the string?

`"When true the HTTP request blocks until the blockchain transaction is confirmed"`

Comment (peterbroadhurst) [reply doc-site/docs/swagger/swagger.yaml:5073]: oh, sorry - I see you're asking about the YAML formatting.

That's just a choice of the YAML serializer when the string exceeds a certain length, to use an indented newline to continue the string

**Pull request merged**